nixos/modules/virtualisation/xen-dom0.nix

```
@@ -146,6 +156,61 @@ in
         "path"
       ]
     )
+    (mkRenamedOptionModule
+      [
+        "virtualisation"
+        "xen"
+        "efi"
+        "bootBuilderVerbosity"
+      ]
+      [
+        "virtualisation"
+        "xen"
+        "boot"
+        "builderVerbosity"
+      ]
+    )
+    (mkRenamedOptionModule
+      [
+        "virtualisation"
+        "xen"
+        "efi"
+        "trace"
+      ]
+      [
+        "virtualisation"
+        "xen"
+        "boot"
+        "trace"
+      ]
+    )
+    (mkRenamedOptionModule
+      [
+        "virtualisation"
+        "xen"
+        "bootParams"
+      ]
+      [
+        "virtualisation"
+        "xen"
+        "boot"
+        "params"
+      ]
+    )
+    (mkRenamedOptionModule
+      [
+        "virtualisation"
+        "xen"
+        "efi"
+        "path"
+      ]
+      [
+        "virtualisation"
+        "xen"
+        "efiBoot"
+        "path"
+      ]
```

```suggestion
      [
        "virtualisation"
        "xen"
        "boot"
        "efi"
        "path"
      ]
```
I really like the new `boot` tree of options, but having `boot`, `efiBoot` and `biosBoot` seems weird. Could you move them to `boot.efi` and `boot.bios` respectively?

---

Done, I agree that's much nicer.